distribution/ddf-common/src/main/resources/bin/ddf.bat

```diff
@@ -49,10 +49,14 @@ SET RC=%ERRORLEVEL%
 REM Check if restart was requested by ddf_on_error.bat
 IF EXIST "%DIRNAME%\restart.jvm" (
     ECHO Restarting JVM...
-    CALL %DDF_HOME%/solr/bin/solr.cmd stop -p %SOLR_PORT%
+    IF "%SOLR_CLIENT%" == "HttpSolrClient" (
+      CALL %DDF_HOME%/solr/bin/solr.cmd stop -p %SOLR_PORT%
+    )
```

Comment:
Could pull lines 52-54 into a subroutine. I haven't  done it before, don't know if it will work as expected, but it would looke something like:

```
:STOP_SOLR
IF "%SOLR_CLIENT%" == "HttpSolrClient" (
       CALL %DDF_HOME%/solr/bin/solr.cmd stop -p %SOLR_PORT%
 )
GOTO :EOF
```

Then you can use `CALL :STOP_SOLR` to stop Solr.

Put the subroutine at the bottom of the file. 
Just before the subroutine is define, add `EXIT /B`